corpus/templates/pages/sig.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block title %}
+{{sig.name}}
+{% endblock %}
+
+{% block content %}
+<section class="hero min-h-[50vh] min-w-screen bg-gradient-to-br from-sky-50 to-blue-300 dark:from-blue-950 dark:to-base-100">
+    <div id="particles-js" class="h-[50vh] w-full"></div>
+    <div class="flex flex-col lg:flex-row gap-20 justify-between mx-auto">
+        {% for society in societies_linked_to_sig %}
+            <a class="my-5" href="{{ society.url }}">
+                {% if society.dark_image %}
+                    <img src="{% static society.image %}" class="padding-10 light-logo h-14 lg:h-24 inline">
+                    <img src="{% static society.dark_image %}" class="padding-10 dark-logo h-14 lg:h-24 inline">
+                    {% else %}
+                    <img src="{% static society.image %}" class="padding-10 h-14 lg:h-24 inline">
```

Comment:
```suggestion
    <div class="flex flex-col lg:flex-row gap-20 justify-between mx-auto items-center">
        {% for society in societies_linked_to_sig %}
            <a href="{{ society.url }}">
                {% if society.dark_image %}
                    <img src="{% static society.image %}" class="padding-10 light-logo h-16 lg:h-28 inline">
                    <img src="{% static society.dark_image %}" class="padding-10 dark-logo h-16 lg:h-28 inline">
                    {% else %}
                    <img src="{% static society.image %}" class="padding-10 h-16 lg:h-28 inline">
```

Added some final styling changes to make the images look larger and correctly in place.